README.md

```diff
@@ -47,26 +52,35 @@ SSB-JS organization require discussion and consent.
 - Projects SHOULD have automated tests for all target platforms.
 - Projects SHOULD have internally consistent contribution guidelines.
 
-### Contributing
+### Contributors
+
+- 'Contributors' are anyone who participates in a project.
+- Contributors MUST aim to foster an open, welcoming, and harassment-free space.
+- Contributors SHOULD recommend changes in the form of patches.
+- Contributors who want to be maintainers SHOULD participate consistently and often.
+
+### Maintainers
 
-- Anyone MUST be able to propose changes to repositories via pull request.
-- Anyone who is not a maintainer MUST NOT have the ability to push commits to repositories.
-- Maintainers MUST have the ability to push commits to every repository.
+- 'Maintainers' are contributors trusted to steward a project and vote.
+- Maintainers MUST have the 'owner' role on GitHub and NPM.
+- Maintainers SHOULD review and merge outstanding patches for the project they maintain.
+- Maintainers MAY review and merge patches for other projects.
 - Maintainers SHOULD NOT merge their own pull requests.
 - Maintainers SHOULD aim to make contributing a fun and simple experience.
 - Maintainers SHOULD merge project pull requests in a fast and friendly manner.
 - Maintainers SHOULD merge project pull requests that are net positive and pass tests.
 - Maintainers SHOULD NOT suggest small improvements on project pull requests.
 - Maintainers SHOULD suggest their improvements by opening their own pull request.
 - Maintainers SHOULD close pull requests that are inactive for more than 30 days.
+- Maintainers SHOULD invite high-quality contributors to become maintainers.
+- Maintainers SHOULD remove anyone who fails to apply this process.
```

Comment:
If I merge my own PR ... I want someone to have a conversation before dropping me plz

Reply:
1 strike and you are out! Boom :-)